tests/bud/base-with-arg/Containerfile2

```diff
@@ -0,0 +1,11 @@
+FROM alpine as build
+ARG CUSTOM_TARGET
```

Comment:
What does this do?

Reply:
Not actually relevant for this test since none of our steps actually use it.